www/artists/dloran/ContentFlow/contentflow.js

```diff
@@ -1,2 +1,2 @@
 /* ContentFlow, version 1.0.2  | (c) 2007 - 2010 Sebastian Kutsch | <http://www.jacksasylum.eu/ContentFlow/> | ContentFlow is distributed under the terms of the MIT license. | (see http://www.jacksasylum.eu/ContentFlow/LICENSE) */
```

Comment:
You're certain this is accurate, correct? I'd assume that, if you downloaded the newest version, it would be a *-_different version_ than the original, but both comments say 1.0.2, suggesting that the version didn't change. If this is the case, what actually caused the issue? And if the version did change, why doesn't the attribution comment reflect this?